pyproject.toml

```diff
@@ -27,7 +27,7 @@ classifiers = [
 python = ">=3.7.0,<4"
 datadog = ">=0.41.0,<1.0.0"
 wrapt = "^1.11.2"
-ddtrace = "1.15.2"
+ddtrace = "<2.0.0"
```

Comment:
There's also a reference to `1.15.2` in the `poetry.lock` file.

Reply:
It seems you'll just need to run `poetry install`, see https://python-poetry.org/docs/basic-usage/#installing-with-poetrylock.

Reply:
Yup, added that already